modules/configuration-annotation-processor/src/main/java/org/apache/ignite/internal/configuration/processor/Processor.java

```diff
@@ -152,6 +152,10 @@ private boolean process0(RoundEnvironment roundEnvironment) {
                     .addModifiers(PUBLIC);
 
             for (VariableElement field : fields) {
+                if (field.getModifiers().contains(STATIC)) {
```

Comment:
I think that `fields` method already returns only non-static members

Reply:
You're right